### ISSUE TITLE
Apply NVDA’s Volume option to NVDA’s sound cues as well

### ISSUE THREAD

**Issue by bhavyashah** [nvaccess/nvda#6756]
Currently, toggling the Volume option present in NVDA’s Voice Settings dialog only brings about a change in the volume of the synthesizer in use by NVDA. It seems unintuitive and logically non-compliant as well as practically confusing for the same option not to apply to the sound cues produced by NVDA. For e.g. toggling NVDA’s volume should also affect NVDA’s focus mode, browse mode, start and exit sounds among other present auditory indications.
Thus, I propose changing NVDA’s volume setting to affect its sound feedback and auditory hints as well. Alternatively, if this doesn’t seem an overkill, a separate slider should be introduce for configuring NVDA’s Sound Volume, because arguably, different speech synthesizers and variants can be more or less loud/soft and thus distinctly different from the constant volumes of NVDA’s sounds.

**Comment by jcsteh** [comment]
Duplicate of #1409. See the discussion there. In short, we need a way to adjust this, but I'm not convinced tying this to the voice volume does make sense, not least because it is "voice", not "audio". #1409 discussed making this optional so it's at least clear what's going on.
